modules/tensor_mechanics/src/materials/ComputeMultipleInelasticStress.C

```diff
@@ -129,6 +134,39 @@ ComputeMultipleInelasticStress::initialSetup()
     else
       mooseError("Model " + models[i] + " is not compatible with ComputeMultipleInelasticStress");
   }
+
+  // Check if tangent calculation methods are consistant. If all models have
+  // TangentOperatorEnum::ELASTIC or tangent_operator is set by the user as elasic, then the tangent
+  // is never calculated: J_tot = C. If FULL and NONE models are present, utilize FULL formulation:
+  // J_tot = (I + J_1 + ... J_N)^-1 C. If PARTIAL and NONE models are present, utilize PARTIAL
+  // formulation: J_tot = J_1 * ... J_N * C. If PARTIAL and FULL models are present, error out.
```

Comment:
I think you have the formulas for combining in the FULL and PARTIAL cases mixed around in this comment.  Also, isn't the FULL formula J_tot = J_1 * C^_1 * ... * J_N * C^-1 * C ?

Reply:
Ah yep. Good points.

Reply:
Yeah, except for the last C^-1: J_tot = J_1 * C^-1 * J_2 * C^-1 * ... J_N